src/custom/hooks/useGetGpApiStatus.ts

```diff
@@ -0,0 +1,39 @@
+import ms from 'ms.macro'
```

Comment:
Please rename the name of this file. It's not a "Api status" . For me api status is UP, DOWN, ...

This is a price strategy.

Reply:
done